web/src/test/java/com/epam/rft/atsy/cucumber/candidate/CandidateApplicationStepDefs.java

```diff
@@ -0,0 +1,160 @@
+package com.epam.rft.atsy.cucumber.candidate;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfAllElementsLocatedBy;
+
+import com.epam.rft.atsy.cucumber.util.DriverProvider;
+import com.epam.rft.atsy.service.domain.CandidateApplicationDTO;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
+public class CandidateApplicationStepDefs {
+  private static WebDriver webDriver = DriverProvider.getDriver();
+  private static final SimpleDateFormat
+      SIMPLE_DATE_FORMAT =
+      new SimpleDateFormat("yyyy. MM. dd. HH:mm:ss");
+
+  private static final String APPLICATIONS_TABLE_ID = "applications_table";
+  private static final String POSITION_NAME = "Fejlesztő";
+  private static final String STATE_TYPE = "Beugró";
+
+  private static final Integer TIME_OUT_FIVE_SECONDS = 5;
+  private static final Integer COLUMN_ZEROTH = 0;
+  private static final Integer COLUMN_FIRST = 1;
+  private static final Integer COLUMN_SECOND = 2;
+  private static final Integer COLUMN_THIRD = 3;
+
+  private static List<CandidateApplicationDTO> candidateApplicationDTOList = null;
+
+
+  @Given("^the user on an existing candidates page$")
+  public void the_user_on_an_existing_candidates_page() throws Throwable {
+    webDriver.get("http://localhost:8080/atsy/secure/candidate/3");
```

Comment:
Maybe you should use constant or property file for this address, or at least for the domain.